examples/Console/TestOtlpExporter.cs

```diff
@@ -66,12 +67,15 @@ private static object RunWithActivitySource(string endpoint, string protocol)
                 otlpExportProtocol = OtlpExportProtocol.Grpc;
             }
 
+            var storagePath = Path.GetTempPath();
+
             // Enable OpenTelemetry for the sources "Samples.SampleServer" and "Samples.SampleClient"
             // and use OTLP exporter.
             using var tracerProvider = Sdk.CreateTracerProviderBuilder()
                     .AddSource("Samples.SampleClient", "Samples.SampleServer")
                     .ConfigureResource(r => r.AddService("otlp-test"))
-                    .AddOtlpExporter(opt =>
+                    .AddOtlpExporterWithPersistentStorage(
```

Comment:
lets leave this example until things get a bit more stable enough to be tested. For now, use the readme.md under the new component to document the usage.

Reply:
@mic-max - Could you add a test instead. It is ok to update the code later to handle specific rpc exceptions. For now you can add a test which confirms that the data is stored offline. (We can do it without using `OpenTelemetry.Extensions.PersistentStorage`).